pymedphys/_trf/decode/trf2csv.py

```diff
@@ -40,29 +42,34 @@ def trf2csv_by_directory(input_directory, output_directory):
         table.to_csv(table_csv_filepath)
 
 
-def trf2csv(trf_filepath, skip_if_exists=False):
-    if not os.path.exists(trf_filepath):
+def trf2csv(trf_filepath, output_directory=None):
+    trf_filepath = pathlib.Path(trf_filepath)
+
+    if not trf_filepath.exists():
         raise Exception("The provided trf filepath cannot be found.")
```

Comment:
Remove this check entirely. Pandas will raise it when the file is being loaded. Also, use of bare "Exception" instead of "FileNotFoundError".